direct/tpm2/clear_test.go

```diff
@@ -0,0 +1,25 @@
+package tpm2
+
+import (
+	"testing"
+
+	"github.com/google/go-tpm/direct/structures/tpm"
+	"github.com/google/go-tpm/direct/transport/simulator"
+)
+
+func TestClear(t *testing.T) {
+
+	thetpm, err := simulator.OpenSimulator()
+	if err != nil {
+		t.Fatalf("could not connect to TPM simulator: %v", err)
+	}
+	defer thetpm.Close()
+
+	clear := Clear{
+		AuthHandle: tpm.RHLockout,
+	}
+
+	if _, err := clear.Execute(thetpm); err != nil {
+		t.Fatalf("Clear failed: %v", err)
+	}
```

Comment:
Is it possible to have some observable affect here? I know TPM2_Clear does multiple things, would it be possible to observe one of them?

Reply:
You could generate a primary key in the storage hierarchy with the same template before and after the clear, and assert the public part changed.

Reply:
Good idea, I like that